src/runtimes/node/utils/entry_file.ts

```diff
@@ -1,23 +1,88 @@
-import { ModuleFormat } from './module_format.js'
+import { basename, extname, resolve } from 'path'
+
+import type { FeatureFlags } from '../../../feature_flags.js'
+
+import { getFileExtensionForFormat, ModuleFileExtension, ModuleFormat } from './module_format.js'
 import { normalizeFilePath } from './normalize_path.js'
 
+export interface EntryFile {
+  contents: string
+  filename: string
+}
+
+const getEntryFileContents = (mainPath: string, moduleFormat: string) => {
+  const importPath = `.${mainPath.startsWith('/') ? mainPath : `/${mainPath}`}`
+
+  if (moduleFormat === ModuleFormat.COMMONJS) {
+    return `module.exports = require('${importPath}')`
+  }
+
+  return `export { handler } from '${importPath}'`
+}
+
+// They are in the order that AWS Lambda will try to find the entry point
+const POSSIBLE_LAMBDA_ENTRY_EXTENSIONS = [ModuleFileExtension.JS, ModuleFileExtension.MJS, ModuleFileExtension.CJS]
+
+// checks if the file is considered a entry-file in AWS Lambda
+export const isNamedLikeEntryFile = (
```

Comment:
Any reason to be using a combination of normal arguments and named parameters here?

Reply:
My logic was that using a named param makes it more readable. So `file` is part of the message `is_namedLikeEntryFile(file, additional_options)`

Same with `conflictsWithEntryFile`, the title relates to the named param  `_conflictsWithEntryFile(srcFiles, additional_options)`

If it is mangled into the options then I would be wondering which of the params is actually `namedLikeEntryFile`.

Reply:
Sounds good!